src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -871,6 +871,7 @@
     <InternalsVisibleToTest Include="Microsoft.CodeAnalysis.Scripting.Destkop.UnitTests" />
     <InternalsVisibleToTest Include="Microsoft.CodeAnalysis.CSharp.Scripting.UnitTests" />
     <InternalsVisibleToTest Include="Microsoft.CodeAnalysis.CSharp.Scripting.Desktop.UnitTests" />
+    <InternalsVisibleToTest Include="Roslyn.Services.Editor.UnitTests" />
```

Comment:
@dotnet/roslyn-compiler can anyone from compiler team can take a look at this change? I added test only IVT so that I can use some internal type in unit tests.